# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/2d/ScaledFontMac.cpp
# Commit: d8c58792b823
# Full Hash: d8c58792b823afb9fe11a1655e6f3e0fcd133510
# Author: Lee Salzman <lsalzman@mozilla.com>
# Date: 2023-04-16 21:18:48
# Regressor Bug: 1821512
# File Overlap Count: 1
# Description:
#   Bug 1821512 - Fix Skia usage for API changes. r=jrmuizel
#   
#   This just tries to address fairly random changes in the Skia API and correct
#   our usage of it in Moz2D and some other places.
#   
# ==============================================================================

diff -r 3427275d765c -r d8c58792b823 gfx/2d/ScaledFontMac.cpp
--- a/gfx/2d/ScaledFontMac.cpp	Sun Apr 16 15:34:50 2023 +0000
+++ b/gfx/2d/ScaledFontMac.cpp	Sun Apr 16 15:34:50 2023 +0000
@@ -164,7 +164,7 @@
 }
 
 SkTypeface* ScaledFontMac::CreateSkTypeface() {
-  return SkCreateTypefaceFromCTFont(mCTFont);
+  return SkMakeTypefaceFromCTFont(mCTFont).release();
 }
 
 void ScaledFontMac::SetupSkFontDrawOptions(SkFont& aFont) {